gradle/verification-metadata.xml

```diff
@@ -1,42 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-
-     Licensed to the Apache Software Foundation (ASF) under one
-     or more contributor license agreements.  See the NOTICE file
-     distributed with this work for additional information
-     regarding copyright ownership.  The ASF licenses this file
-     to you under the Apache License, Version 2.0 (the
-     "License"); you may not use this file except in compliance
-     with the License.  You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing,
-     software distributed under the License is distributed on an
-     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-     KIND, either express or implied.  See the License for the
-     specific language governing permissions and limitations
-     under the License.
-
--->
-<!--
-   This file declares the dependency verification metadata for dependencies used by Groovy.
-   When you upgrade a dependency, verification is likely to fail so you will need to update
-   it.
-
-   If, for whatever reason, verification fails for you, DO NOT DISABLE IT but follow the
-   troubleshooting guide at https://docs.gradle.org/current/userguide/dependency_verification.html
-
-   This build uses PGP verification and SHA512 checksum verification for unsigned artifacts:
-
-   ./gradlew -write-verification-metadata sha512,pgp -export-keys <tasks>
-
-   Then check the diff, remove the unnecessary entries (for old dependencies) and make sure
-   the new ones are correct.
-
-   Do not remove this header.
```

Comment:
:x: This says "Do not remove this header" and the PR removes the header. It shows that the file hasn't been reviewed, when it should ;)

Reply:
Good catch ;-)